packages/a11y-speak/src/domReady.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Specify a function to execute when the DOM is fully loaded.
+ *
+ * @param {Function} callback A function to execute after the DOM is ready.
+ *
+ * @returns {void}
+ */
+const domReady = function( callback ) {
+  if ( document.readyState === 'complete' || ( document.readyState !== 'loading' && !document.documentElement.doScroll ) ) {
```

Comment:
There should be a space after the `!` exclamation:

>Any `!` negation operator should have a following space.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing

We may need to update our ESLint configuration for `space-unary-ops` options:

http://eslint.org/docs/rules/space-unary-ops#options